src/tools/osdmaptool.cc

```diff
@@ -403,14 +408,24 @@ int main(int argc, const char **argv)
       cout << "No pools available" << std::endl;
       goto skip_upmap;
     }
+    int rounds = 0;
+    struct timespec round_start;
+    int r = clock_gettime(CLOCK_MONOTONIC, &round_start);
+    assert(r == 0);
+    do {
```

Comment:
I'm indenting the code here.

Reply:
yes, or move the body to a helper function